internal/ambient/types.go

```diff
@@ -0,0 +1,17 @@
+package ambient
+
+// DaprTrustBundle represents all keys provided by Dapr.
+type DaprTrustBundle struct {
+	CertKey      string
+	CertChain    string
+	TrustAnchors string
+}
+
+// ToMap converts a DaprTrustBundle struct to map[string]string.
+func (d *DaprTrustBundle) ToMap() map[string]string {
```

Comment:
@mcruzdev it might be better to consume the Dapr dependencies to get these values, as was done before.. right? to avoid redefining variables that can change on the dapr side.